index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>NotesHD | Hetav Desai</title>
+  </head>
+  <body>
+    <script src="theme.js" type="text/javascript"></script>
+    <div class="container">
+      <section class="left">
+        <div class="header">
+          <h2 class="notes-list-heading">Notes HD</h2>
+          <div class="theme-switch-wrapper">
+            <label class="theme-switch" for="checkbox">
+              <input type="checkbox" id="checkbox" />
```

Comment:
`checkbox` value for an id is too generic...it might conflict later. Have something more specific.